services/app-web/src/components/FileUpload/FileUpload.tsx

```diff
@@ -51,20 +51,16 @@ export const FileUpload = ({
     uploadFile,
     scanFile,
     deleteFile,
-    onLoadComplete,
+    onFileItemsUpdate,
     ...inputProps
 }: FileUploadProps): React.ReactElement => {
-    const [loadingStatus, setLoadingStatus] = useState<
-        null | 'UPLOADING' | 'COMPLETE'
-    >(null)
     const [fileItems, setFileItems] = useState<FileItemT[]>(initialItems || [])
     const fileInputRef = useRef<FileInputRef>(null) // reference to the HTML input which has files
 
+    // update fileItems in parent
     React.useEffect(() => {
-        if (loadingStatus !== 'UPLOADING') {
-            onLoadComplete({ files: fileItems })
-        }
-    }, [fileItems, loadingStatus, onLoadComplete])
+        onFileItemsUpdate({ fileItems })
```

Comment:
🔍 This change is the actual bug fix. Basically making the onUpdate fire every time. Before it would not figure if file was still uploading